Team54/src/main/java/ISA/Team54/security/config/WebSecurityConfig.java

```diff
@@ -91,7 +91,7 @@ public void configure(WebSecurity web) throws Exception {
 		// TokenAuthenticationFilter ce ignorisati sve ispod navedene putanje
 		web.ignoring().antMatchers(HttpMethod.POST, "/auth/login");
 		web.ignoring().antMatchers(HttpMethod.GET, "/", "/webjars/**", "/*.html", "/favicon.ico", "/**/*.html",
-				"/**/*.css", "/**/*.js");
+				"/**/*.css", "/**/*.js","/api/patientBySurnameAndName/*","/api/allPatients/");
```

Comment:
Ako se ignorisu ovi endpointi znaci da ce im svi imati pristup. Je l to samo privremeno za proveru rada?

Reply:
Samo privremeno za proveru